src/CommunityToolkit.Datasync.Server.CosmosDb/CommunityToolkit.Datasync.Server.CosmosDb.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\CommunityToolkit.Datasync.Server.Abstractions\CommunityToolkit.Datasync.Server.Abstractions.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Cosmos" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
+    <PackageReference Include="Newtonsoft.Json" />
```

Comment:
Why is Newtonsoft required?

Reply:
Hi Adrian, this is a dependency of the Cosmos Sdk even though I've elected to use System.Text.Json I believe Newtonsoft is still required under the hood.

Reply:
I've attempted to remove the dependency and disable the check but the program crashes on the instantiation of the CosmosClient.

Reply:
I’ll open a bug with the Cosmos folks on Monday.  Newtonsoft should be defined as a transitive dependency in that case.

Reply:
Hi @adrianhall, there are already multiple threads on this issue. We can mark the library to not required the dependency directly but then it would be up to the consumer to install it in their api project, it would generate an error at build time. We would still need it as a dependency then for the sample project. It's a bit of a mess, they seem to be one foot in one foot out.